apps/labs/posts/intel-oneapi-msvc-spin-scipy.md

```diff
@@ -0,0 +1,174 @@
+---
+title: 'Enhancing Developer Experience at SciPy - Intel oneAPI/MSVC Support and Migrating to spin'
+published: February 21, 2025
+authors: [gagandeep-singh]
+description: 'Highlights the work done to improve developer experience at SciPy, specifically on supporting Intel oneAPI/MSVC and spin'
+category: [Developer workflows]
+featuredImage:
+  src: /posts/intel-oneapi-msvc-spin-scipy/scipy_logo_img_featured.png
+  alt: 'SciPy logo'
+hero:
+  imageSrc: /posts/intel-oneapi-msvc-spin-scipy/scipy_logo_img_hero.png
+  imageAlt: 'SciPy logo'
+---
+
+In this blog post, we’ll discuss the improvements we made to enhance the developer experience at SciPy. Our work focused on two key areas:
```

Comment:
```suggestion
Over the past year, we worked on enhancing the development experience of SciPy. We focused on two key areas:
```

The start feels a little abrupt, hence this suggestion for a softer introduction.